Tests/SwiftyJSONTests/NestedJSONTests.swift

```diff
@@ -39,4 +39,36 @@ class NestedJSONTests: XCTestCase {
         ]
         XCTAssertNotNil(try? nestedFamily.rawData())
     }
+    
+    func testArrayJSON() {
+        let arr: [JSON] = ["a", 1, ["b", 2]]
+        let json = JSON(arr)
+        XCTAssertEqual(json[0].string, "a")
+        XCTAssertEqual(json[2,1].int, 2)
+    }
+    
+    func testDictionaryJSON() {
+        let json: JSON = ["a": JSON("1"), "b": JSON([1, 2, "3"]), "c": JSON(["aa": "11", "bb": 22])]
+        XCTAssertEqual(json["a"].string, "1")
+        XCTAssertEqual(json["b"].array!, [1, 2, "3"])
+        XCTAssertEqual(json["c"]["aa"].string, "11")
+    }
+    
+    func testNextedJSON() {
+        let inner = JSON.init([
```

Comment:
Swift code completion not works some times when typing `JSON(` but wokrs well with `JSON.init` 😊